tools/db_crashtest.py

```diff
@@ -129,6 +131,8 @@ def finalize_and_sanitize(src_params):
             dest_params["db"]):
         dest_params["use_direct_io_for_flush_and_compaction"] = 0
         dest_params["use_direct_reads"] = 0
+    if dest_params["partition_filters"] == 1:
+        dest_params["index_type"] == 2
```

Comment:
should it be '=' instead of '=='?

Reply:
should we also set use_block_based_filter to false here just in case someone added it later?